helm/values.yaml

```diff
@@ -18,11 +18,19 @@ container:
   readOnlyRootFilesystem: true
   allowPrivilegeEscalation: false
   apiGatewayUrl: http://mine-support-api-gateway.default
-  restartPolicy: Always
+  restartPolicy: Always  
+  
 ingress:
   serviceName: mine-support
   endPoint: mine-support
   server: defradev.com
 replicas: 1
 minReadySeconds: 5
 cookiePassword: passwordpasswordpasswordpasswordpassword
+
+livenessProbe:          
```

Comment:
there's a lot of whitespace after the `livenessProbe:` on this line

Reply:
Fixed